@xen-orchestra/backups/merge-worker/cli.mjs

```diff
@@ -34,6 +35,7 @@ const main = Disposable.wrap(async function* main(args) {
     for (let i = 0; i < 10; ++i) {
       const entries = await handler.list(CLEAN_VM_QUEUE)
       if (entries.length !== 0) {
+        entries.sort()
```

Comment:
Why?

Reply:
because the replaced code used lodash.min instead of just shift